uvsim.py

```diff
@@ -58,6 +58,7 @@ def store(self, filename):
         """
         Store the contents of memory to a file
         """
+
         # Find the index of the last non-empty opcode
         last_non_empty_index = -1
         for i in range(len(self.mem) - 1, -1, -1):
```

Comment:
@jordanpaxman  pro tip in future: you can iterate backwards with `your_list[::-1]`, youd have to enumerate that array to get the index again but it would get you what you want.